app/code/Magento/Sales/Helper/Guest.php

```diff
@@ -227,11 +225,14 @@ private function setGuestViewCookie($cookieValue)
      */
     private function loadFromCookie($fromCookie)
     {
+        if (!is_string($fromCookie)) {
```

Comment:
Hello @xmav, thanks for you review, I checked you solution, and I think which is better to leave !is_string($fromCookie), because method `$this->cookieManager->getCookie(self::COOKIE_NAME)` returns string or null, so we must leave the old behavior for method `loadFromCookie()` and `throw new InputException(__($this->inputExceptionMessage))` if $fromCookie is null.

What you think?

Thank you